Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -935,6 +968,42 @@ theorem mem_closure_one_iff_norm {x : E} : x ∈ closure ({1} : Set E) ↔ ‖x
 theorem closure_one_eq : closure ({1} : Set E) = { x | ‖x‖ = 0 } :=
   Set.ext fun _x => mem_closure_one_iff_norm
 
+section Instances
+
+instance [SeminormedAddGroup E] : ContinuousENorm E where
+/- XXX: continuous_nnnorm errors with
```

Comment:
I presume `continuous_nnnorm'` (with prime) errors with this? The reason is that you didn't define a multiplicative version of this instance.

Reply:
The unprimed version errors - but using `continuous_nnnorm'` fixes it (both in the additive and multiplicative version).

Reply:
(There are further errors further below, see the main comment.)